core/src/main/java/com/scalar/db/api/LikeExpression.java

```diff
@@ -0,0 +1,152 @@
+package com.scalar.db.api;
+
+import com.google.common.base.MoreObjects;
+import com.scalar.db.io.TextColumn;
+import java.util.Objects;
+import java.util.regex.Pattern;
+import javax.annotation.Nonnull;
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+public class LikeExpression extends ConditionalExpression {
+  private static final String DEFAULT_ESCAPE_CHAR = "\\";
+  @Nonnull private final String escape;
```

Comment:
[trivial] IIUC, instance variables are regarded as non-null by default and we can remove this `@Nonnull`...?

Reply:
Fixed in 9654df0.